docs/configuration.md

```diff
@@ -149,7 +149,7 @@ For example, if you have 10 ETH available in your wallet on the exchange and `tr
 
 #### Amend last stake amount
 
-Assuming we have a `balance=1000` (USDT), `stake_amount=400`, and `max_open_trades=3`.
+Assuming we have the tradable balance of 1000 USDT (`balance=1000`), `stake_amount=400`, and `max_open_trades=3` (This example assumes `tradable_balance_ratio=1`).
```

Comment:
What is `balance=1000` here? We do not have such a setting... And adding tradable_balance_ratio only makes things overcomplicated for the reader...

For me,
"Assuming we have the tradable balance of 1000 USDT, `stake_amount=400`, and `max_open_trades=3`."
would be enough.

It's tradable balance, whatever it's defined.

Reply:
Let's remove "(This example assumes `tradable_balance_ratio=1`).", it's not true -- because this example does not assume this, it's actual for any tradable_balance_ratio if tradable balance is 1000, for any reason with any "full" balance and any tradable_balance_ratio, if they combination gives tradable balance equal to 1000.

Reply:
well it's not wrong either - if i set this to 0.5, then the example would be wrong.

I've removed it now however.

Reply:
Hmm, wallet is 2000 USDT, tradable_balance_ratio=0.5, then tradable balance is 1000 USDT and the example is actual...

Reply:
well yes - we can change it in many ways to either have it correct or wrong - but i think as it is now it's correct (?) ...